### ISSUE TITLE
Support for Tailwind 3+

### ISSUE THREAD

**Issue by tdwesten** [josemarluedke/tailwindcss-ember-power-select#42]
Hi, 

I got some warnings when using your plugin with Tailwind 3.0.18; 
```
warn - The `content` option in your Tailwind CSS configuration is missing or empty.
warn - Configure your content sources or your generated CSS will be missing styles.
warn - https://tailwindcss.com/docs/content-configuration
```

And i got no styling at all, I think because the missing content defs. 

Do you have any plans to support Tailwind 3+?

**Comment by tdwesten** [comment]
Adding add `savelist` to the `tailwind.config.js` helps;
```
  safelist: [
    {
      pattern: /^ember-power-select-/,
    },
  ],
```

**Comment by tdwesten** [comment]
CC: https://twitter.com/adamwathan/status/1492514071119384585?s=21

**Comment by zion** [comment]
@tdwesten were you able to resolve this? I tried adding that safelist pattern, but that did not resolve my issue.

**Comment by tdwesten** [comment]
@zion yes, I have it working in this project; https://github.com/tdwesten/shareyourtraveltips-web/blob/main/app/styles/tailwind.config.js

**Comment by zion** [comment]
For anyone struggling with peer dependency issues regarding tailwindcss version when installing packages, I forked this project and updated tailwind to v3. Heres a [PR](https://github.com/josemarluedke/tailwindcss-ember-power-select/pull/54) for it
